plugin/trino-opensearch/src/main/java/io/trino/plugin/opensearch/OpenSearchConfig.java

```diff
@@ -326,6 +327,19 @@ public OpenSearchConfig setIgnorePublishAddress(boolean ignorePublishAddress)
         return this;
     }
 
+    public boolean isProjectionPushdownEnabled()
+    {
+        return projectionPushDownEnabled;
+    }
+
+    @Config("opensearch.projection-pushdown-enabled")
+    @ConfigDescription("Read only required fields from a row type")
+    public OpenSearchConfig setProjectionPushdownEnabled(boolean projectionPushDownEnabled)
+    {
```

Comment:
Add a `good-first-issue` follow-up issue in the community to port this feature in `trino-elasticsearch` as well if it applies.